pom.xml

```diff
@@ -155,6 +155,11 @@
       <artifactId>jitsi-util</artifactId>
       <version>${jitsi-desktop.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.jitsi</groupId>
+      <artifactId>jitsi-webrtcvadwrapper</artifactId>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
Do you want `1.0-20200414.144800-2` here?

Reply:
Oh, that's duplicate

Reply:
oh I will remove it

Reply:
something went wrong on cherry pick/rebase, thanks for noticing

Reply:
https://github.com/jitsi/jigasi/pull/263